samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/ui/IssuanceTestScreen.kt

```diff
@@ -0,0 +1,44 @@
+package com.android.identity.testapp.ui
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.android.identity.issuance.WalletApplicationCapabilities
+import com.android.identity.testapp.issuance.WalletServerProvider
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.datetime.Clock
+
+@Composable
+fun IssuanceTestScreen() {
+    LazyColumn(
+        modifier = Modifier.padding(8.dp)
+    ) {
+        item {
+            TextButton(
+                onClick = {
+                    CoroutineScope(Dispatchers.Default).launch {
+                        val provider = WalletServerProvider(
+                            {
+                                WalletApplicationCapabilities(
+                                    generatedAt = Clock.System.now(),
+                                    androidKeystoreAttestKeyAvailable = true,
+                                    androidKeystoreStrongBoxAvailable = true,
+                                    androidIsEmulator = false
```

Comment:
Maybe add `expect val isEmulator: Boolean` to `com.android.identity.testapp.Platform` and have it be true if on Android emulator or iOS simulator?

Reply:
Done